nc/source/physics/CollisionSystem.cpp

```diff
@@ -2,130 +2,185 @@
 #include "debug/Profiler.h"
 #include "directx/math/DirectXCollision.h"
 
+namespace
+{
+    DirectX::BoundingSphere EstimateBoundingVolume(const nc::physics::ColliderSoA::CenterExtentPair& volumeData, const DirectX::XMMATRIX* transform)
```

Comment:
Did we ever come up with a plan to extend this for shapes like the castle wall example we had discussed? (Where a sphere is not appropriate)